prow/tide/blockers/blockers.go

```diff
@@ -70,16 +73,56 @@ func (b Blockers) GetApplicable(org, repo, branch string) []Blocker {
 }
 
 // FindAll finds issues with label in the specified orgs/repos that should block tide.
-func FindAll(ghc githubClient, log *logrus.Entry, label, orgRepoTokens string) (Blockers, error) {
+func FindAll(ghc githubClient, log *logrus.Entry, label string, orgRepoTokensByOrg map[string]string, splitQueryByOrg bool) (Blockers, error) {
+	queries := make(map[string]string, len(orgRepoTokensByOrg))
+	for org, query := range orgRepoTokensByOrg {
+		if splitQueryByOrg {
+			queries[org] = blockerQuery(label, query)
+		} else {
+			if queries[""] != "" {
+				queries[""] += " "
+			}
+			queries[""] += blockerQuery(label, query)
+		}
+	}
+	if !splitQueryByOrg {
+		// Deduplicate common query elements
+		queries[""] = strings.Join(sets.NewString(strings.Split(queries[""], " ")...).List(), " ")
+	}
+
+	var issues []Issue
+	var errs []error
+	var lock sync.Mutex
+	var wg sync.WaitGroup
+
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
 	defer cancel()
-	issues, err := search(
-		ctx,
-		ghc,
-		log,
-		blockerQuery(label, orgRepoTokens),
-	)
-	if err != nil {
+	for org, query := range queries {
+		org, query := org, query
+		wg.Add(1)
+
+		go func() {
+			defer wg.Done()
+			result, err := search(
+				ctx,
+				ghc,
+				org,
+				log,
+				query,
+			)
```

Comment:
nit: liebreaking above looks a bit excessive

Reply:
Which linebreaking exactly do you mean?

Reply:
`result, err := search(ctx, ghc, org, og, query)